tests/Xamarin.Forms-Performance-Integration/Xamarin.Forms.Performance.Integration.sln

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
This file shouldn't exist. Instead, the .csproj should be added to the top level Xamarin.Android-Tests.sln file.

Reply:
Good catch, updated.